wrapper/build.gradle.kts

```diff
@@ -79,7 +79,7 @@ dependencies {
     testImplementation("org.apache.poi:poi-ooxml:5.3.0")
     testImplementation("org.slf4j:slf4j-simple:2.0.13")
     testImplementation("com.fasterxml.jackson.core:jackson-databind:2.17.1")
-    testImplementation("com.amazonaws:aws-xray-recorder-sdk-core:2.18.1")
+    testImplementation("com.amazonaws:aws-xray-recorder-sdk-core:2.18.2")
```

Comment:
Should we update our sdk versions in our other build.gradle files as well? If you search for "software.amazon.awssdk" and "aws-xray" in the project's `*.kts` files you should be able to find them all